lib/ansible/modules/windows/win_domain_controller.ps1

```diff
@@ -24,6 +22,9 @@ Function Write-DebugLog {
 
     Write-Debug $msg
 
+    $log_path = $null
+    $log_path = Get-AnsibleParam -obj $params -name "log_path"
```

Comment:
This is a change to how the module behaves, should be in a separate PR.

Reply:
Why this is considered a behavior change?
 it just inserts the vars of that function into it.

Reply:
Because before this wasn't done ?

Reply:
before, the same line was a global var outside the function that is why I don't think it should be considered a behavior change.

If this was a module it could have a lot of repercussions

Reply:
I agree this shouldn't be in this PR, keep behavioral changes to a separate PR.